docs/04-operation-guides/troubleshooting/smgt-01-hb.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Helm Broker troubleshooting
+---
+
```

Comment:
For troubleshooting topics, I suggest three standardized headlines:

## Symptom
## Cause
## Remedy

Reply:
It would be good to pass this info to other TWs so that all troubleshooting docs are consistent.